packages/app/src/components/Body.tsx

```diff
@@ -57,27 +57,42 @@ export const Body = ({ yOffset, data, config }: Props) => {
    * and an error message is returned if the plot is aggregated.
    * @returns A promise that resolves to the generated alt text.
    */
-  async function generateAltText(): Promise<string> {
+  async function generateAltText(): Promise<AltText> {
+
+    /**
+     * Converts an error message to an AltText object.
+     *
+     * @param err - The error message to convert.
+     * @returns The AltText object with the error message as the long description, short description, and technique description.
+     */
+    function errToAltText(err: string): AltText {
```

Comment:
What's the reasoning behind this? Could you throw an error and catch it in the calling function instead?

Reply:
Good point. This should be addressed now; I'm using errors instead. It's also hiding the "Show More" button when an error occurs.